src/Thenewboston/Common/Api/NodeConfigService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Threading.Tasks;
+using Thenewboston.Common.Api;
+using Thenewboston.Common.Http;
+
+namespace Thenewboston.Validator.Api
+{
+    public class NodeConfigService : INodeConfigService
+    {
+
+        private readonly IHttpRequestSender _requestSender;
+
+        public NodeConfigService(IHttpRequestSender requestSender)
+        {
+            _requestSender = requestSender;
+        }
+
+        public async Task<T> GetConfigAsync<T>()
```

Comment:
I see what you were trying to do, but I think it leaves us prone to errors and confusion. It's better to have two separate classes in Bank and Validator, and specify the return type.